tests/sql/test_aggs.py

```diff
@@ -21,6 +21,7 @@ def test_aggs_sql():
                 col("values").min().alias("min"),
                 col("values").max().alias("max"),
                 col("values").count().alias("count"),
+                col("values").count_distinct().alias("count_distinct"),
```

Comment:
change the dataframe up so that the result of count and count distinct are different

Reply:
Updated.